packages/mux-video/src/base.ts

```diff
@@ -167,6 +167,11 @@ export class MuxVideoBaseElement extends CustomVideoElement implements IMuxVideo
         this.updateLogo();
       }
     });
+
+    this.addEventListener('mux-check-disable-pseudo-ended', (event: Event): void => {
+      (event as CustomEvent<{ shouldDisable: boolean }>).detail.shouldDisable =
+        this.hasAttribute('disable-pseudo-ended');
+    });
```

Comment:
this should be removed and we should add a get / set disablePseudoEnded prop in this class...
then in playback-core we can access that prop